### DISCUSSION TITLE
Since touch method is deprecated

### DISCUSSION THREAD

**Discussion by firmansyahrosid** [webdriverio/webdriverio#12449] [Q&A]
hi, because xcuitest driver deprecates some touch related methods. Is there any other option Wdio can use besides driver.touchon to tap the screen without using elements?

**Comment by christian-bromann** [comment]
@firmansyahrosid we recommend using the [`actions`](https://webdriver.io/docs/api/browser/action/) command moving forward.

**Reply by firmansyahrosid** [reply]
can we use for mobile app ?

**Reply by christian-bromann** [reply]
Yes, that should work.
